eli5/__init__.py

```diff
@@ -44,3 +44,10 @@
         pass
     else:
         raise
+
+
+try:
+    from .lightgbm import explain_weights_lightgbm
```

Comment:
These imports were mostly for singledispatch discovery to work - explain_prediction_xgboost is not there as well. But it indeed makes sense to re-export more of these functions; I'll fix that, thanks for the catch!